bluepysnap/simulation.py

```diff
@@ -100,19 +113,19 @@ def time_units(self):
     @property
     def conditions(self):
         """Access to the conditions dictionary for this simulation."""
-        return self.config.get("conditions", {})
+        return getattr(self.to_libsonata, "conditions", None)
```

Comment:
With this change, `conditions` would return `None` instead of `{}` when the attribute is missing.
Is it intentional? In theory it could break code using it.

Reply:
Yeah, `libsonata` returns an object (`libsonata._libsonata.Conditions`) if it was defined and I felt it would be counter-intuitive to return empty dictionary by default.

About breaking changes, this would break the code using it, anyway. We're releasing a new major version, so I am not too worried about small breaking changes. 

I agree, I should mention these in the changelog. I'll update it.